addon/server.js

```diff
@@ -196,7 +211,8 @@ export default class Server {
   */
   loadFactories(factoryMap) {
     // Store a reference to the factories
-    this._factoryMap = factoryMap;
+    this._factoryMap = this._factoryMap || {};
+    this._factoryMap = Object.assign(this._factoryMap, factoryMap);
```

Comment:
Just saw the failure. If you click `details` next to the red Travis failure you can see the build logs. Search `"not ok"` and you'll see the failure, `Object.assign is not a function`. This is because we don't have a polyfill and it's not supported in Phantomjs.

[Elsewhere](https://github.com/samselikoff/ember-cli-mirage/blob/2c31ad15a46e90b357a83b6896c6774fa42c6488/addon/serializer.js#L3) we just use _.assign from lodash, so let's do that here.

```js
import _assign from 'lodash/object/assign';
...
_assign(currentFactoryMap, factoryMap)
```